src/AdminOnlyProperty/wwwroot/App_Plugins/AdminOnlyProperty/lang/da.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<language alias="da" intName="Danish" localName="Dansk" lcid="" culture="da-DK">
+    <area alias="adminOnlyProperty">
+        <key alias="packageName">Admin Kun Ejendom</key>
```

Comment:
```suggestion
        <key alias="packageName">Admin Only Property</key>
```

Don't really think the package name should be translated...

Reply:
@greystate Awww 😿

Reply:
@greystate I chatted with @LottePitcher, she says that you are correct. 👍